src/dfx/src/commands/sns/download.rs

```diff
@@ -0,0 +1,29 @@
+//! Code for the command line `dfx sns import`
+use crate::lib::error::DfxResult;
+use crate::lib::info::replica_rev;
+use crate::lib::nns::install_nns::download_sns_wasms;
+use crate::Environment;
+use std::path::PathBuf;
+
+use clap::Parser;
+use tokio::runtime::Runtime;
+
+/// Downloads the SNS canister WASMs
+#[derive(Parser)]
+pub struct SnsDownloadOpts {
+    /// IC commit of SNS canister WASMs to download
+    #[clap(long)]
```

Comment:
Clap supports this - will have to add the `"env"` feature for clap in Cargo.toml
```suggestion
    #[clap(long, env("DFX_IC_COMMIT"))]
```